third_party/nxp/k32w0_sdk/sdk_fixes/patch_k32w_sdk.sh

```diff
@@ -11,16 +11,13 @@ SOURCE_DIR=$(cd "$(dirname "$SOURCE")" >/dev/null 2>&1 && pwd)
 cp ./third_party/nxp/k32w0_sdk/sdk_fixes/gpio_pins.h "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/openthread/reed/bm/
 cp ./third_party/nxp/k32w0_sdk/sdk_fixes/pin_mux.c "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/openthread/enablement/
 
-#TODO internal: https://jira.sw.nxp.com/browse/MCB-2678
-cp ./third_party/nxp/k32w0_sdk/sdk_fixes/SecLib.h "$NXP_K32W061_SDK_ROOT"/middleware/wireless/framework/SecLib/
-
 # TODO internal: https://jira.sw.nxp.com/browse/MCB-2675
 cp -r ./third_party/nxp/k32w0_sdk/sdk_fixes/app_dual_mode_low_power.h "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/hybrid/ble_ot/lped_ble_wuart/ble_802_15_4_common/
 cp -r ./third_party/nxp/k32w0_sdk/sdk_fixes/app_dual_mode_switch.h "$NXP_K32W061_SDK_ROOT"/boards/k32w061dk6/wireless_examples/hybrid/ble_ot/lped_ble_wuart/ble_802_15_4_common/
 
-cp -r ./third_party/nxp/k32w0_sdk/sdk_fixes/OtaUtils.c "$NXP_K32W061_SDK_ROOT"/middleware/wireless/framework/OtaSupport/Source/
-
-patch -d "$NXP_K32W061_SDK_ROOT"/middleware/wireless/bluetooth/host/interface -p1 <"$SOURCE_DIR/patch_ble_utils_h.patch"
+patch -d "$NXP_K32W061_SDK_ROOT"/middleware/wireless/framework/SecLib --binary -p1 <"$SOURCE_DIR/patch_SecLib_h.patch"
+patch -d "$NXP_K32W061_SDK_ROOT"/middleware/wireless/framework/OtaSupport/Source --binary -p1 <"$SOURCE_DIR/patch_OtaUtils_c.patch"
```

Comment:
nit: did we consider using same line endings (usually unix-style) instead of needing `-binary`?

In hindsight, naming things `patch_foo.patch` stutters. We could just say `foo.patch`

Reply:
Solved it: removed the binary, and fixed the naming. thanks!